packages/model-viewer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google/model-viewer",
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
Thanks for all the dependency updates! I don't believe any of this will be breaking changes though, correct? So no need for a major version bump. Also, we do version bumps of our packages when we prepare a release, not during regular PRs, so you can revert this.